scripts/seedUser.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const csvParser = require('csv-parser');
+const { Client } = require('pg');
+const moment = require('moment');
+
+
+console.log('Getting postgres URL');
+const env = process.env.NODE_ENV || 'development';
+// eslint-disable-next-line import/no-dynamic-require
+const config = require(`${__dirname}/../config/config.js`)[env];
+
+if (!config.url) {
+  console.error('Postgres url not found. Please ensure you set one in your env.');
+  process.exit(1);
+}
+
+const filePath = process.argv[2];
+
+const client = new Client({
```

Comment:
We may be able to use the sequelize seeders file method option by using the `queryInterface.sequelize.query` to run the sql query or queryInterface.sequelize.bulkInsert` passing in the data read from the file without the need for a client

https://sequelize.org/master/manual/migrations.html#running-seeds

Reply:
This file is not making use of sequelize. I'm using the raw pg library.